parachains/runtimes/bridge-hubs/bridge-hub-rococo/Cargo.toml

```diff
@@ -89,14 +89,17 @@ bridge-runtime-common = { path = "../../../../bridges/bin/runtime-common", defau
 
 [dev-dependencies]
 static_assertions = "1.1"
+asset-test-utils = { path = "../../assets/test-utils"}
```

Comment:
@acatangiu 
looks very good on the first sight, but need to check deeply.

just one very nit, could we re-export `use asset_test_utils::{mock_open_hrmp_channel, ExtBuilder};` in `bridge-hub-test-utils`?  I've planned to move `asset-test-utils` maybe one day one level up from assets as `sp-test-utils`

Reply:
done